frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/doclet/FrankMethod.java

```diff
@@ -28,4 +30,12 @@
 	FrankType[] getParameterTypes();
 	boolean isVarargs();
 	FrankAnnotation getAnnotationInludingInherited(String name) throws FrankDocException;
+	String getJavaDoc();
+	String getJavaDocIncludingInherited() throws FrankDocException;
+	String getDefaultValueFromJavadoc();
```

Comment:
Kan je dit niet beter getDefaultValue noemen?

Reply:
In de IbisDoc annotatie zit ook een default value. In deze pull request worden nog zowel IbisDoc annotaties als JavaDocs ondersteund. Daarom wilde ik duidelijk maken dat het hier om de default uit de JavaDocs gaat.